sdk/src/main/java/com/mapbox/maps/renderer/MapboxRenderThread.kt

```diff
@@ -120,23 +123,30 @@ internal class MapboxRenderThread : Choreographer.FrameCallback {
     }
     if (!surface.isValid) {
       Logger.w(TAG, "EGL was configured but surface is not valid.")
-      postPrepareRenderFrame()
+      // give system a bit of time and try rendering again hoping surface will be valid now
+      postPrepareRenderFrame(delayMillis = RETRY_DELAY_MS)
```

Comment:
Do we know existing cases when this happens?

Reply:
no